actors/builtin/power/power_actor.go

```diff
@@ -544,6 +545,19 @@ func (a Actor) OnEpochTickEnd(rt Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
 	return &adt.EmptyValue{}
 }
 
+func (a Actor) MinerNominalPowerMeetsConsensusMinimum(rt Runtime, _ *adt.EmptyValue) bool {
+	var st State
+	rt.State().Readonly(&st)
+	store := adt.AsStore(rt)
+	minerAddr := rt.Message().Receiver()
+
+	gtMinSize, err := st.minerNominalPowerMeetsConsensusMinimum(store, minerAddr)
+	if err != nil {
+		rt.Abortf(exitcode.ErrIllegalState, "Failed to check miner %v meets min required size: %v", minerAddr, err)
+	}
+	return gtMinSize
+}
+
```

Comment:
Please do not add an actor method unless it needs to be called by some other actor. Extra-actor code (including EPost validation) will access this state directly.